File: 09_Canvas/app/src/main/java/com/example/a09_canvas/MainActivity.java
package com.example.a09_canvas;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Context;
import android.content.Intent;
import android.graphics.Bitmap;
import android.graphics.Canvas;
import android.graphics.Color;
import android.graphics.Paint;
import android.graphics.Rect;
import android.os.Bundle;
import android.util.AttributeSet;
import android.util.TypedValue;
import android.view.View;
import android.widget.Button;
import android.widget.ImageView;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);


        Button bNextPage = findViewById(R.id.NextPage);
        bNextPage.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent intent = new Intent(MainActivity.this, SpriteActivity.class);
                startActivity(intent);
            }
        });
    }
}

File: 04_Interface_Elements/app/src/main/java/com/example/a04_interface_elements/MainActivity.kt
package com.example.a04_interface_elements

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button

class MainActivity : AppCompatActivity() {
    private lateinit var pressedButton: Button
    private lateinit var nextPageButton: Button

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        pressedButton = findViewById(R.id.PressedButton)
        pressedButton.setOnClickListener {
            pressedButton.setBackgroundResource(R.drawable.pressed_button)
        }
        nextPageButton = findViewById(R.id.NextPage)
        nextPageButton.setOnClickListener {
            val intent = Intent(this@MainActivity, PressedOrReleasedActivity::class.java)
            startActivity(intent)
        }
    }
}